hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/ProvidesHoodieConfig.scala

```diff
@@ -124,7 +124,7 @@ trait ProvidesHoodieConfig extends Logging {
     val preCombineField = hoodieCatalogTable.preCombineKey.getOrElse("")
 
     val hiveStylePartitioningEnable = Option(tableConfig.getHiveStylePartitioningEnable).getOrElse("true")
```

Comment:
SQL functionality relies heavily on partition management and a encoded `partitionPath` is the key for support.